ReleaseNotes-2.9

```diff
@@ -38,6 +38,7 @@ UI:
    * A basic and expert mode for adding repositories is available.
    * obsrepositories:/ source_path can be added easily using 'use project repositories' checkbox.
    * Only `apt-deb` and `rpm-md` are allowed as repo_type. Other values will be replaced by `rpm-md`.
+   * Added autocompletion for binary packages names coming from backend.
```

Comment:
[nitpick] I would drop `from backend` here:
* It's harder to understand (speaking from the user perspective)
* The user doesn't care about where the information come from.